src/libraries/System.Security.Cryptography.Xml/ref/System.Security.Cryptography.Xml.cs

```diff
@@ -173,7 +179,7 @@ public sealed partial class EncryptionPropertyCollection : System.Collections.IC
         [System.Runtime.CompilerServices.IndexerName("ItemOf")]
         public System.Security.Cryptography.Xml.EncryptionProperty this[int index] { get { throw null; } set { } }
         public object SyncRoot { get { throw null; } }
-        object System.Collections.IList.this[int index] { get { throw null; } set { } }
+        object? System.Collections.IList.this[int index] { get { throw null; } set { } }
```

Comment:
is this actually nullable if we throw when null is added?

Reply:
same comment for every IList

Reply:
As far as I know, it needs to match the `IList` signature.